gpytorch/kernels/arc_kernel.py

```diff
@@ -0,0 +1,181 @@
+        \end{equation}
+
+    and the produt between dimensions
+
+    .. math::
+        \begin{equation}
+        k_{i}(\mathbf{x}, \mathbf{x^{'}}) = \sigma^{2}\exp(-\frac{1}{2}d_{i}(\mathbf{x}, \mathbf{x^{'}}))^{2}
+        \end{equation}
+    .. note::
+
+        This kernel does not have an `outputscale` parameter. To add a scaling
+        parameter, decorate this kernel with a
+        :class:`gpytorch.kernels.ScaleKernel`.
+        When using with an input of `b x n x d` dimensions, decorate this
+        kernel with :class:`gpytorch.kernel.ProductStructuredKernel , setting
+        the number of dims, `num_dims to d.`
+    .. note::
+
+        This kernel does not have an ARD lengthscale option.
+
+    Args:
+        :attr:`base_kernel` (gpytorch.kernels.Kernel):
+            The euclidean covariance of choice. Default: `MaternKernel(nu=2.5)`
+        :attr:`ard_num_dims` (int):
+            The number of dimensions to compute the kernel for. The kernel has
+            two parameters which are individually defined for each dimension.
+            Default: `None`.
+        :attr:`angle_prior` (Prior, optional):
+            Set this if you want to apply a prior to the period angle
+            parameter.  Default: `None`.
+        :attr:`radius_prior` (Prior, optional):
+            Set this if you want to apply a prior to the lengthscale parameter.
+            Default: `None`.
+
+    Attributes:
+        :attr:`radius` (Tensor):
+            The radius parameter. Size = `*batch_shape  x 1`.
+        :attr:`angle` (Tensor):
+            The period angle parameter. Size = `*batch_shape  x 1`.
+
+    Example:
+        >>> x = torch.randn(10, 5)
+        >>> # Non-batch: Simple option
+        ... base_kernel = gpytorch.kernels.MaternKernel(nu=2.5)
+        >>> base_kernel.raw_lengthscale.requires_grad_(False)
+        >>> covar_module = gpytorch.kernels.ProductStructureKernel(
+                gpytorch.kernels.ScaleKernel(
+                    ArcKernel(base_kernel,
+                              angle_prior=gpytorch.priors.GammaPrior(0.5,1),
+                              radius_prior=gpytorch.priors.GammaPrior(3,2),
+                              ard_num_dims=x.shape[-1])),
+                num_dims=x.shape[-1])
+        >>> covar = covar_module(x)
+        >>> print(covar.shape)
+        >>> # Now with batch
+        >>> covar_module = gpytorch.kernels.ProductStructureKernel(
+                gpytorch.kernels.ScaleKernel(
+                    ArcKernel(base_kernel,
+                              angle_prior=gpytorch.priors.GammaPrior(0.5,1),
+                              radius_prior=gpytorch.priors.GammaPrior(3,2),
+                              ard_num_dims=x.shape[-1])),
+                num_dims=x.shape[-1])
+        >>> covar = covar_module(x
+        >>> print(covar.shape)
+    """
+
+    def __init__(self,
+                 base_kernel=MaternKernel(nu=2.5),
```

Comment:
I think the doc building won't like having an actual object be the default argument here, so we'll probably need this to be a required argument

Reply:
Ok, I delete it then and leave it as required